source/simulator/assemblers/newton_stokes.cc

```diff
@@ -449,7 +450,7 @@ namespace aspect
                 data.local_rhs(i) += (
                                        // RHS of momentum eqn: - \int 2/3 eta R, div v
                                        - 2.0 / 3.0 * eta
-                                       * prescribed_dilation->dilation[q]
+                                     * prescribed_dilation->dilation[q]
```

Comment:
You will probably have to run the indentation script to make this change go away again.